### ISSUE TITLE
Babel throwing error about exportDefaultFrom even after adding the plugin

### ISSUE THREAD

**Issue by FrediArro** [babel/babel#11392]
## Bug Report

- [ ] I would like to work on a fix!

**Current Behavior**
I created my porject with CRA.
I tried to import React-Navigation to my project. After the import, babel started throwing me an error for "@babel/plugin-proposal-export-default-from" plugin every time i tried to run my code. So I added it and also updated my babel configuration, but it still keeps throwing me this error. What am I missing?
```
SyntaxError: .../node_modules/react-native-gesture-handler/DrawerLayout.js: Support for the experimental syntax 'exportDefaultFrom' isn't currently enabled (30:8):

  28 | const SETTLING = 'Settling';
  29 | 
> 30 | export type PropType = {
     |        ^
  31 |   children: any,
  32 |   drawerBackgroundColor?: string,
  33 |   drawerPosition: 'left' | 'right',

Add @babel/plugin-proposal-export-default-from (https://git.io/vb4yH) to the 'plugins' section of your Babel config to enable transformation.
```
**Input Code**
- REPL or Repo link if applicable:

https://github.com/FrediArro/babel-error

**Expected behavior/code**
No errors

**Babel Configuration (babel.config.js, .babelrc, package.json#babel, cli command, .eslintrc)**
- Filename: `package.json`

``` js
"babel": {
    "plugins": [
      "@babel/plugin-proposal-export-default-from"
    ]
  }
```

**Environment**

```
 System:
    OS: Linux 5.3 Ubuntu 18.04.4 LTS (Bionic Beaver)
  Binaries:
    Node: 12.16.1 - ~/.nvm/versions/node/v12.16.1/bin/node
    Yarn: 1.21.1 - /usr/bin/yarn
    npm: 6.13.4 - ~/.nvm/versions/node/v12.16.1/bin/npm
  npmPackages:
    @babel/plugin-proposal-export-default-from: ^7.8.3 => 7.8.3 
    @babel/plugin-proposal-export-namespace-from: ^7.8.3 => 7.8.3 
```
**Possible Solution**

**Additional context/Screenshots**
Add any other context about the problem here. If applicable, add screenshots to help explain.

**Comment by JLHwung** [comment]
@FrediArro Thanks for your report, especially for the reproduction repo.

I am afraid I have to close it because it is Duplicate of #7954, which will be fixed by #8080. Since both Flow and TypeScript supports type exports now, I will see if I can revive #8080.

**Comment by JLHwung** [comment]
Duplicate of #7954
